src/Iris.php

```diff
@@ -3,38 +3,42 @@
 namespace ElevenMiles\Iris;
 
 
+
 use ElevenMiles\Iris\WordPress\Admin\AdminPanel;
-use ElevenMiles\Iris\Helper\IrisHelper;
 use ElevenMiles\Iris\IrisConvert\IrisConvert;
 use ElevenMiles\Iris\WordPress\API;
 use ElevenMiles\Iris\WordPress\CLI\Register;
 use ElevenMiles\Iris\WordPress\Config;
 use ElevenMiles\Iris\WordPress\Scripts;
 
 include_once(ABSPATH . 'wp-admin/includes/plugin.php');
+require_once(ABSPATH . 'libraries/action-scheduler/action-scheduler.php');
 
 /**
  * Class Iris
- * 
- * For debugging: Debug::debug($image_meta);
- * 
  */
 
 class Iris
 {
-    // move to .env
-    const WP_API_NAMESPACE = 'wyke/v1';
+
     /**
-     * Iris constructor 
+     * Iris constructor.
      */
+
     public function __construct()
     {
-        // 
         new AdminPanel();
+        add_action('plugins_loaded', [__CLASS__, 'checkActionSchedulerPlugin']);
```

Comment:
What does `checkActionSchedulerPlugin` do? Is that adding a admin notice as such using the `is_plugin_active` check? If so, perhaps worth making `checkActionSchedulerPlugin` return the output of the check so that it can be used below rather than re-running the check in another place?